activesupport/test/cache/stores/redis_cache_store_test.rb

```diff
@@ -193,6 +195,23 @@ def test_write_expires_at
       end
     end
 
+    def test_increment
```

Comment:
```suggestion
    def test_increment_ttl
```

`test_increment` is already defined in `cache/behaviors/cache_increment_decrement_behavior.rb`, so this is replacing it.